src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -103,7 +103,7 @@ class PalantirDockerPlugin implements Plugin<Project> {
                         group = 'Docker'
                         description = "Tags Docker image with tag '${tagName}'"
                         workingDir dockerDir
-                        commandLine 'docker', 'tag', '--force=true', ext.name, computeName(ext.name, tagName)
+                        commandLine 'docker', 'tag', ext.name, computeName(ext.name, tagName)
```

Comment:
I don't think you'll have that issue in 1.10: https://docs.docker.com/engine/deprecated/#f-flag-on-docker-tag:9c0d3b2efa16c5c274427efa73f212c9. But since most people are probably on 1.9, we shouldn't merge it just yet